lib/view/widget/text/pictron_password_text_field.dart

```diff
@@ -0,0 +1,13 @@
+import 'package:flutter/material.dart';
+import 'package:pictron/view/widget/text/pictron_base_text_field.dart';
+
+class PictronPasswordTextField extends PictronBaseTextField {
+  PictronPasswordTextField({
+    Key key,
+    String placeholder,
+  }) : super(
+    key: key,
+    placeholder: placeholder,
+    obscureText: true,
+  );
+}
```

Comment:
Let's try to add Line feeds at the end of the files. See the following [page](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline).